testbed/core/models.py

```diff
@@ -75,9 +113,9 @@ def get_json_ld(self):
                 "https://swicg.github.io/activitypub-data-portability/lola.jsonld",
             ],
             "type": "Person",
-            "id": f"https://example.com/users/{self.username}",
-            "preferredUsername": self.username,
-            "name": self.username,
+            "id": f"https://example.com/users/{self.user.username}",
+            "preferredUsername": self.user.username,
```

Comment:
Actors are separate from users - there can be multiple per user 
So the id can't be the same for both
the preferredUsername doesn't need to be the user's username - it could be "Alice" or "Bob" or "Dave" (destination) and "Sarah" (source)